src/managers/auth.py

```diff
@@ -229,12 +229,20 @@ def add_acl(
         ]
 
         if resource_type == "TOPIC":
-            command += [f"--topic={resource_name}"]
+            if len(resource_name) > 3 and resource_name.endswith("*"):
```

Comment:
**question**: Is there a need for more validation using a regex to ensure that the resource is indeed 3 chars and a *, and not some other value?
Something like `^[a-z]{3,}\*$`

Reply:
I agree with Marc. Given if it is a fairly simple use case, I would also prioritize readability. I would use regex when one has more complex patterns with a nice comment explaining what the regex does. Although powerful, I find regex one of the most exoteric syntaxes